<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AKG" failures="0" tests="1" name="testRunner.TestNGRunner" time="5.194" errors="1" timestamp="13 Jan 2023 11:42:37 GMT" skipped="0">
  <testcase classname="testRunner.TestNGRunner" name="runScenario" time="5.194">
    <error message="&quot;user login into application with username &quot;newtest&quot; and password &quot;Test@123&quot;&quot; matches more than one step definition:
  &quot;^user login into application with (.+) and (.+)$&quot; in stepDefinitions.LoginwithDataTablesSteps2.user_login_into_application_with_and(java.lang.String,java.lang.String)
  &quot;^user login into application with username &quot;([^&quot;]*)&quot; and password &quot;([^&quot;]*)&quot;$&quot; in stepDefinitions.LoginSteps.user_login_into_application_with_username_something_and_password_something(java.lang.String,java.lang.String)" type="io.cucumber.core.runner.AmbiguousStepDefinitionsException">
      <![CDATA[io.cucumber.core.runner.AmbiguousStepDefinitionsException: "user login into application with username "newtest" and password "Test@123"" matches more than one step definition:
  "^user login into application with (.+) and (.+)$" in stepDefinitions.LoginwithDataTablesSteps2.user_login_into_application_with_and(java.lang.String,java.lang.String)
  "^user login into application with username "([^"]*)" and password "([^"]*)"$" in stepDefinitions.LoginSteps.user_login_into_application_with_username_something_and_password_something(java.lang.String,java.lang.String)
at io.cucumber.core.runner.CachingGlue.findStepDefinitionMatch(CachingGlue.java:402)
at io.cucumber.core.runner.CachingGlue.stepDefinitionMatch(CachingGlue.java:370)
at io.cucumber.core.runner.Runner.matchStepToStepDefinition(Runner.java:183)
at io.cucumber.core.runner.Runner.createTestStepsForPickleSteps(Runner.java:163)
at io.cucumber.core.runner.Runner.createTestCaseForPickle(Runner.java:146)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:74)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:129)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:129)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRunner.TestNGRunner -->
